server/gokbg3/grails-app/controllers/org/gokb/rest/UsersController.groovy

```diff
@@ -0,0 +1,189 @@
+package org.gokb.rest
+
+import grails.converters.JSON
+import grails.gorm.transactions.Transactional
+import grails.plugin.springsecurity.annotation.Secured
+import org.gokb.cred.User
+
+@Secured(['ROLE_USER', 'IS_AUTHENTICATED_FULLY'])
+@Transactional(readOnly = true)
+class UsersController {
+
+  static namespace = 'rest'
+
+  def userProfileService
+
+  @Secured(value = ['ROLE_ADMIN', 'IS_AUTHENTICATED_FULLY'], httpMethod = 'GET')
+  def show() {
```

Comment:
Default search action elsewhere is `index()`.

Reply:
changed